### PR TITLE
nxos_linkagg: removing unsupported parameter "provider" and "timeout" from sanity test

### PR DESCRIPTION
**Pull request by nkshrishail**
##### SUMMARY
Removing unsupported parameter "provider" and "timeout" from the sanity test for nxos_linkagg test module.

##### ISSUE TYPE
- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by chrisvanheuveln** [comment]
shipit

**Review by NilashishC — CHANGES_REQUESTED** [review]
@nkshrishail removing `provider` would break integration tests, because we are still running against `ansible_connection=local` [here](https://github.com/ansible/ansible/blob/devel/test/integration/targets/nxos_linkagg/tasks/nxapi.yaml#L30).

**Comment by nkshrishail** [comment]
@NilashishC , right, there is another PR coming in soon which will remove the support for local nxapi plugin and will also remove any references of local and also of provider. This one can be merged after the upcoming PR gets merged.

**Comment by mikewiebe** [comment]
@nkshrishail Is this PR still valid since you have the larger PR that removes provider for all tests?  If it is still valid you need to make sure to remove the nxapi local block under `tasks/nxapi.yaml` for this test.

**Comment by nkshrishail** [comment]
@mikewiebe yeah, it is, I have actually left out these files in the larger PR.

**Review by NilashishC — APPROVED** [review]

**Pull request merged**
